proto/batch_plan.proto

```diff
@@ -30,6 +30,11 @@ message RowSeqScanNode {
   }
   // If along with `batch_limit`, `chunk_size` will be set.
   ChunkSize chunk_size = 6;
+  message Limit {
+    uint64 limit = 1;
+  }
+  // The pushed down `batch_limit`. Max rows needed to return.
+  Limit limit = 7;
```

Comment:
```suggestion
  optional uint64 limit = 7;
```

Reply:
Should this be following:
```
message Limit {
  uint32 offset = 1;
  uint 32 limit = 2;
}
```

So that we can optimize following sql:
```sql
select * from t limit 10, 20;
```

Reply:
After a two-phase limit optimization. It will not generate an offset for the scan. BTW used offset would cause a wrong result as we have multiple partition.